bindings/go/main.go

```diff
@@ -100,7 +100,10 @@ func LoadTrustedSetupFile(trustedSetupFile string) CKZGRet {
 		panic("error reading trusted setup")
 	}
 	ret := C.load_trusted_setup_file(&settings, fp)
-	C.fclose(fp)
+	if C.fclose(fp) != 0 {
+		C.free_trusted_setup(&settings)
```

Comment:
This is slightly surprising because we are freeing the trusted setup without checking the retval of `load_trusted_setup_file`.